### ISSUE TITLE
[broken] artisan infyom.publish:layout

### ISSUE THREAD

**Issue by illuminate3** [InfyOmLabs/laravel-generator#430]
The vendor package: core-templates was never loaded thus error when trying to get templates.

quick hack: down load and upload that package without composer ... lol
This causes a app/http/routes/web.php error.

hack II: copy routes directory from the 5.4 location.

The command to generate this: config/infyom/laravel_generator.php never fired ... no error ... just missing file.

hack III: copy from package directory and paste into config directory

I did read later instructions and saw how to add the missing package into composer .json ...

HomeController doesn't add namespace.

Still testing to see what else I'll find.

**Comment by whereisjefe** [comment]
I ran into the same issue and it was caused by me skipping a step on these [instructions](http://labs.infyom.com/laravelgenerator/docs/5.4/installation#existing-projects). However, if you are using the boilerplate i can't speak to that.

**Comment by illuminate3** [comment]
I went through the instructions several times to make sure I didn't skip step.

I'm not complaining but wanted to post back what got me up and running in case other people had similar problems and thought you all would like the feed back.

It's working for me know and I'm trying to learn all the commands. This is pretty nifty!
I used mysql workbench to setup my tables and relationships and then wanted a quick and easy way to generate all the basic files from the database. Until now everything I saw on google was forward generations rather than reverse generations.
